google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -55,6 +55,24 @@
  *   <li>The default service address (bigtable.googleapis.com) and default port (443) are used.
  *   <li>Credentials are acquired automatically through Application Default Credentials.
  *   <li>Retries are configured for idempotent methods but not for non-idempotent methods.
+ *   <li>These are the default timeouts used Bigtable operations.
+ *       <ul>
+ *         For non-streaming operations i.e. {@link #readRowSettings()}, {@link
+ *         #sampleRowKeysSettings()}, {@link #checkAndMutateRowSettings()} {@link
+ *         #mutateRowSettings()}, {@link #bulkMutateRowsSettings()} {@link
+ *         #readModifyWriteRowSettings()}:
+ *         <li>Default timeout for {@link RetrySettings#getInitialRpcTimeout()} is 20_000ms.
+ *         <li>Default timeout for {@link RetrySettings#getMaxAttempts()} is 20_000ms.
```

Comment:
timeout for max attempts as a duration?

Reply:
My apologies, It was a typo from my end it should be `getMaxRpcTimeout`.

Reply:
I would collapse the rpc timeout min/max/multiplier settings into a single constant rpc timeout value in the docs